security/registries.adoc

```diff
@@ -0,0 +1,92 @@
+[[security-registries]]
+= Registries
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:sourcedir: ../
+
+toc::[]
+
+[[security-registries-where-do-your-containers-come-from]]
+== Where Do Your Containers Come From?
+
+As discussed, there are tools you can use to scan and track the contents of your
```

Comment:
So this is referencing the "Scanning Container Content" topic? Should it link to it? Or even be included with that content?